sdk/scheduler/src/main/java/com/mesosphere/sdk/api/EndpointsResource.java

```diff
@@ -81,22 +79,24 @@ public EndpointsResource setCustomEndpoint(String name, EndpointProducer endpoin
     @GET
     public Response getEndpoints(@QueryParam("format") String format) {
         try {
-            JSONObject endpoints = new JSONObject();
+            List<String> endpoints = new ArrayList<>();
+
             // Custom values take precedence:
             for (Map.Entry<String, EndpointProducer> entry : customEndpoints.entrySet()) {
-                endpoints.put(entry.getKey(), entry.getValue().getEndpoint());
+                endpoints.add(entry.getKey());
             }
             // Add default values (when they don't collide with custom values):
             for (Map.Entry<String, JSONObject> endpointType :
                     getDiscoveryEndpoints(
                             serviceName,
                             stateStore.fetchTasks(),
                             isNativeFormat(format)).entrySet()) {
-                if (!endpoints.has(endpointType.getKey())) {
-                    endpoints.put(endpointType.getKey(), endpointType.getValue());
+                if (!endpoints.contains(endpointType.getKey())) {
+                    endpoints.add(endpointType.getKey());
                 }
             }
-            return Response.ok(endpoints.toString(), MediaType.APPLICATION_JSON).build();
+
+            return Response.ok(new JSONArray(endpoints).toString(), MediaType.APPLICATION_JSON).build();
```

Comment:
The old way just output a map of endpoint name -> value? I'm fine with this, but it seems to have changed the purpose of that command.